docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -33,6 +33,12 @@ The main features of the Media Controller API include:
 
   To [receive and handle incoming commands](#receive_custom_commands) in the server, use the `addCommandListener()` method.
 
+- Sending and processing search requests
+
+  You can use the media controller client to [send the search requests](#sending-search-request) with the `sendSearchRequest()` method.
+
+  You can use the media controller server to [receive the search requests](#receiving-search-request) using `setSearchRequestListener()` method, and return `RequestReply` object to the client who sent the request.
```

Comment:
Suggest rephrasing to:

You can use the media controller server to [receive the search requests](#receiving-search-request) using the `setSearchRequestListener()` method, and return the `RequestReply` object to the client that sent the request.